src/messages/MessageElement.cpp

```diff
@@ -132,6 +134,31 @@ void ImageElement::addToContainer(MessageLayoutContainer &container,
     }
 }
 
+PrettyImageElement::PrettyImageElement(ImagePtr image, int padding,
+                                       QColor background,
+                                       MessageElementFlags flags)
+    : MessageElement(flags)
+    , image_(image)
+    , padding_(padding)
+    , background_(background)
+{
+}
+
+void PrettyImageElement::addToContainer(MessageLayoutContainer &container,
+                                        MessageElementFlags flags)
+{
+    if (flags.hasAny(this->getFlags()))
+    {
+        auto size = QSize(this->image_->width() * container.getScale(),
+                          this->image_->height() * container.getScale());
```

Comment:
Can be simplified.
`QSize(this->image_->width(), this->image_->height()) * container.getScale()`

Reply:
Changed in 936ae06